exo-go/features/run/run.feature

```diff
@@ -0,0 +1,66 @@
+Feature: running Exosphere applications
+
+  As an Exosphere developer
+  I want to have an easy way to run an application on my developer machine
+  So that I can test my app locally.
+
+  Rules:
+  - run "exo run" in the directory of your application to run it
+  - this command boots up all the services of the application
+
+
+  Scenario: booting a functioning Exosphere application
+    Given a running "simple" application
+    Then my machine is running the services:
+      | NAME |
+      | web  |
+
+
+  Scenario: booting a complex Exosphere application
+    Given a running "running" application
+    Then my machine is running ExoCom
+    And my machine is running the services:
+      | NAME  |
+      | web   |
+      | users |
+    # And ExoCom uses this routing:
```

Comment:
@kevgo It looks like the `SERVICE_ROUTES` is already in JSON. Are you talking about `>-`? Because that just indicates that the next line is a string. This isn't something we write but is generated by `docker-compose`.